job_dsl/release.groovy

```diff
@@ -0,0 +1,27 @@
+library "rpc-gating@${RPC_GATING_BRANCH}"
+common.shared_slave(){
+  stage("Configure Git"){
+    common.configure_git()
+  }
+  stage("Release"){
+    withCredentials([
+      string(
+        credentialsId: 'rpc-jenkins-svc-github-pat',
+        variable: 'PAT'
+      ),
+      string(
+        credentialsId: 'mailgun_hughsaunders_endpoint',
+        variable: 'MAILGUN_ENDPOINT'
```

Comment:
Its currently my domain, if account with user/pass in passwordsafe. No domain verified because I didn't want to use my personal domain. That does mean that every recipient must be added to the test recipients list. 

I realise that account isn't optimal, it would be simple to change accounts if someone comes up with a better one.